lambda_uploader/config.py

```diff
@@ -17,11 +17,12 @@
 
 REQUIRED_PARAMS = {u'name': basestring, u'description': basestring,
                    u'region': basestring, u'handler': basestring,
-                   u'role': basestring, u'timeout': int, u'memory': int}
+                   u'role': basestring, u'timeout': int, u'memory': int,
+                   u'vpc': dict}
 
 DEFAULT_PARAMS = {u'requirements': [], u'publish': False,
                   u'alias': None, u'alias_description': None,
-                  u'ignore': [], u'extra_files': []}
+                  u'ignore': [], u'extra_files': [], u'vpc': {}}
```

Comment:
Can we set the default to None and then test for a none type. I would prefer that over a try catch.